### PR TITLE
sprint progress: working on unit tests for friendship work

### PR DESCRIPTION
**Pull request by fm1539**

### CODE REVIEW

File: src/utils/mock-db.ts [hunk 1]
```diff
@@ -31,4 +33,126 @@ export async function findUser(email: string, existing: boolean) {
...
+}
+
+export async function getFriendshipStatus(user1Email: string, user2Email: string, exists: boolean) {
+  const { readQuery, session } = getFriendStatusSetup(user1Email, user2Email, exists)
+  let results = {};
+  return new Promise(async (resolve, reject) => {
+    try {
+      const readResult = await session.run(readQuery, { user1Email, user2Email })
+      readResult.records.forEach((record) => {
+        console.log("record", record);
```
Comment (iao233) [inline src/utils/mock-db.ts]: Remove console log

File: src/utils/mock-db.ts [hunk 2]
```diff
@@ -31,4 +33,126 @@ export async function findUser(email: string, existing: boolean) {
...
+        console.log("recordfields", record["_fields"]);
```
Comment (iao233) [inline src/utils/mock-db.ts]: Remove console log

File: src/controllers/friends.ts [hunk 3]
```diff
@@ -8,113 +8,93 @@ const addFriend = (req: Request, res: Response) => {
...
 
-const sendFriendRequest = (req: Request, res: Response, next: NextFunction) => {
+const sendFriendRequest = async (req: Request, res: Response, next: NextFunction) => {
   const { user1Email, user2Email } = req.body;
 
-  const db = new DB();
-
-  db.createFriendRequest(user1Email, user2Email)
-    .then((value) => {
-      console.log(value);
```
Comment (iao233) [inline src/controllers/friends.ts:18]: Remove console log

File: src/controllers/friends.ts [hunk 4]
```diff
@@ -8,113 +8,93 @@ const addFriend = (req: Request, res: Response) => {
...
   next: NextFunction
 ) => {
   const { user1Email, user2Email } = req.body;
 
-  const db = new DB();
-console.log('34');
+  const db = new DB()
+  try {
+  const value = await db.createFriendship(user1Email, user2Email);
+  console.log("VALUE in CONT", value);
```
Comment (iao233) [inline src/controllers/friends.ts]: Remove console log

File: src/index.ts [hunk 5]
```diff
@@ -15,14 +15,15 @@ const app: Express = express()
 app.use(express.json())
 
 app.use((req, res, next) => {
+  console.log("HEADERS SET")
```
Comment (iao233) [inline src/index.ts]: Remove

**Review by iao233 — CHANGES_REQUESTED** [review]

File: src/controllers/friends.ts (hunk 3 above)
Comment (fm1539) [reply src/controllers/friends.ts:18]: This one is showing from the removed version, i already removed it

**Review by iao233 — APPROVED** [review]

**Pull request merged**